<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WriterWindowClass</class>
 <widget class="QWidget" name="WriterWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WriterWindow</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
      <item>
       <widget class="QPushButton" name="saveAllButton">
        <property name="text">
         <string>Save to file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="descriptionLE"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Act:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="actLE"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add idea</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="acceptButton">
        <property name="text">
         <string>Accept idea</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="developButton">
        <property name="text">
         <string>Develop ideas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save idea</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
